website/content/en/highlights/2021-11-12-event-trottle-transform.md

```diff
@@ -15,9 +15,9 @@ badges:
 
 We've released a new `throttle` transform that provides a user the ability to throttle the throughput of specific event streams.
 
-Large spikes in data volume can frequently overwhelm a service, which is especially common in log data. Previously, users lacked the necessary tooling to control throughput, such as setting quota for users and user groups utilizing Vector, which not only can cause spike in costs, but also negatively impact downstream services due to the increased load. 
+Large spikes in data volume can frequently overwhelm a service, which is especially common in log data. Previously, users lacked the necessary tooling to control throughput, such as setting a limit for users and user groups utilizing Vector, which not only can cause spike in costs, but also negatively impact downstream services due to the increased load. 
```

Comment:
@barieom @binarylogic I suspect we should change the component docs as well to not use quota if we're reserving that for another purpose in the Vector space

Reply:
👍 I'm fine with using the term quota. I think it does apply here. Otherwise, agreed, we should update both places.